backend/controllers/prisonerProfile/prisonerVisits.ts

```diff
@@ -3,20 +3,18 @@ import { hasLength, formatName, putLastNameFirst, sortByDateTime } from '../../u
 import generatePagination from '../../shared/generatePagination'
 import { DATE_TIME_FORMAT_SPEC, MOMENT_TIME } from '../../../common/dateHelpers'
 
+export const VISIT_TYPES = [
```

Comment:
hard code types rather than go to Prison API.  Only two options, they have never changed and we now want different text for the two options instead of what is in the database

Reply:
Are they values that are likely to change in the future?